.github/workflows/MultiXscale:compile_lammps.yml

```diff
@@ -0,0 +1,26 @@
+name: MultiXscale-compile_lammps
+run-name: Compile new Plugin OBMD
+on:
+  push:
+    paths:
+      - '.github/**'
+      - 'src/OBMD/**'
+      - 'examples/OBMD/**'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: eessi/github-action-eessi@v3
+    - name: set up EasyBuild
+      run: |
+        module load EasyBuild/4.9.4
+        module load EESSI-extend/2023.06-easybuild
+        export EASYBUILD_TRACE=1
+        eb --show-config
+        echo "Get EasyConfig"
+        wget https://raw.githubusercontent.com/laraPPr/lammps/refs/heads/OBMD_compile_in_CI/.github/workflows/LAMMPS_OBMD-foss-2023a-software-commit.eb
+        eb LAMMPS_OBMD-foss-2023a-software-commit.eb --software-commit ${{ github.sha }} --include-easyblocks-from-commit 3d295a926318468e3e8fdd99fdcfea02cd796403
+        module avail LAMMPS
+        module load LAMMPS/${{ github.sha }}-foss-2023a-kokkos-dev_OBMD
+        ls /home/runner/eessi/versions/2023.06/software/linux/x86_64/amd/zen3/software/LAMMPS/${{ github.sha }}-foss-2023a-kokkos-dev_OBMD
+        mpirun -np 4 lmp -in /home/runner/eessi/versions/2023.06/software/linux/x86_64/amd/zen3/software/LAMMPS/${{ github.sha }}-foss-2023a-kokkos-dev_OBMD/examples/OBMD/in.OBMD
```

Comment:
I suspect there are not really 4 cores, 4 virtual but only 2 physical. You can either
```
--use-hwthread-cpus
```
or 
```
--oversubscribe
```